frontend/lib/components/NavBar/components/NavItems/components/LanguageDropDown/hooks/useLanguageHook.ts

```diff
@@ -47,18 +51,18 @@ export const useLanguageHook = (): {
 
     // Set language to default if no language found in localStorage
     setCurrentLanguage(languages[0]);
-    localStorage.setItem('selectedLanguage', languages[0].id);
+    localStorage.setItem("selectedLanguage", languages[0].id);
```

Comment:
You need to configure your vscode linter for single quotes instead of double quotes to avoid this kind of changes upon saving the files you're modifying